File: RentalApp.API/Models/Ad.cs
using System;
using System.Collections.Generic;

namespace RentalApp.API.Models
{
    public class Ad
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public float Price { get; set; }
        public string City { get; set; }
        public string Facilities { get; set; }
        public DateTime AvailableFrom { get; set; }
        public DateTime AvailableTill { get; set; }
        public ICollection<Photo> Photos { get; set; }

    }
}

File: RentalApp-SPA/src/app/_models/Ad.ts
import { Photo } from './Photo';
import {Moment} from 'moment';
export interface Ad {
    id: number;
    name: string;
    price: number;
    city: string;
    availablefrom: Date;
    availabletill: Date;
    photoUrl: string;
    facilities?: string;
    photos?: Photo[];
}
